WooCommerce/Classes/ViewRelated/Orders/OrderDetailsViewController.swift

```diff
@@ -466,6 +537,20 @@ private extension OrderDetailsViewController {
         StoresManager.shared.dispatch(action)
     }
 
+    func syncTracking(onCompletion: ((Error?) -> Void)? = nil) {
+        let action = ShipmentAction.synchronizeShipmentTrackingData(siteID: viewModel.order.siteID,
+                                                                    orderID: viewModel.order.orderID) { error in
+            if let error = error {
+                DDLogError("⛔️ Error synchronizing tracking: \(error.localizedDescription)")
+                onCompletion?(error)
+                return
+            }
+            onCompletion?(nil)
```

Comment:
We should add a `orderShipmentsLoaded` Tracks call here (similar to L565 in `syncNotes`). I will send you details on Slack